pkg/s3-proxy/server/tls_test.go

```diff
@@ -0,0 +1,226 @@
+package server
```

Comment:
Can you add the build flag?
```golang
//go:build unit
```
Thanks

Reply:
Will do.

Reply:
Fixed in d5ffbfd40066d5bec7f97c9d4bed1d4f6d3ca4a4